cdc/http_handler.go

```diff
@@ -260,6 +280,100 @@ func (s *Server) handleChangefeedQuery(w http.ResponseWriter, req *http.Request)
 	writeData(w, resp)
 }
 
+// handleChangefeeds dispatch the request to the specified handleFunc according to the request method.
+func (s *Server) handleChangefeeds(w http.ResponseWriter, req *http.Request) {
+	if req.Method == http.MethodGet {
+		s.handleChangefeedsList(w, req)
+		return
+	}
+}
+
+func (s *Server) handleChangefeedsList(w http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodGet {
+		writeError(w, http.StatusBadRequest, cerror.ErrSupportPostOnly.GenWithStackByArgs())
```

Comment:
`req.Method != http.MethodGet` return `ErrSupportPostOnly` ?

Reply:
thanks. I will correct it.